README.md

```diff
@@ -1,14 +1,22 @@
-
-zfs-replicate.sh
-================
-
-Simple script to replicate zfs volumes between hosts (or between pools on the same host) via incremental snapshots.
+# zfs-replicate
+Bash script to automate ZFS Replication
+Forked from https://github.com/aaronhurt/zfs-replicate
```

Comment:
Remove forked text.

Reply:
The README.md in the PR still says the script is forked from _this_ location. When the PR is merged it will no longer be forked.

Reply:
That’s fine.

Reply:
I'll change it once it's merged. It's  not right to say it's a fork on the source.